contracts/test/OperatorsRegistry.1.t.sol

```diff
@@ -894,48 +894,48 @@ contract OperatorsRegistryV1Tests is OperatorsRegistryV1TestBase, BytesGenerator
         assert(operator.keys == 10);
     }
 
-    function testRemoveHalfValidatorsEvenCaseAsOperator(bytes32 _name, uint256 _firstAddressSalt) public {
-        address _firstAddress = uf._new(_firstAddressSalt);
-        vm.startPrank(admin);
-        uint256 index = operatorsRegistry.addOperator(string(abi.encodePacked(_name)), _firstAddress);
-
-        bytes memory tenKeys = genBytes((48 + 96) * 10);
-
-        vm.stopPrank();
-        vm.startPrank(_firstAddress);
-        operatorsRegistry.addValidators(index, 10, tenKeys);
-        vm.stopPrank();
-        vm.startPrank(admin);
-        uint256[] memory operators = new uint256[](1);
-        uint32[] memory limits = new uint32[](1);
-        operators[0] = index;
-        limits[0] = 10;
-        operatorsRegistry.setOperatorLimits(operators, limits, block.number);
-        vm.stopPrank();
-        vm.startPrank(_firstAddress);
-
-        OperatorsV2.Operator memory operator = operatorsRegistry.getOperator(index);
-        assert(operator.keys == 10);
-
-        uint256[] memory indexes = new uint256[](5);
-
-        indexes[0] = 8;
-        indexes[1] = 6;
-        indexes[2] = 4;
-        indexes[3] = 2;
-        indexes[4] = 0;
-
-        vm.expectEmit(true, true, true, true);
-        emit RemovedValidatorKey(index, LibBytes.slice(tenKeys, 0, 48));
-        operatorsRegistry.removeValidators(index, indexes);
-        operator = operatorsRegistry.getOperator(index);
-        assert(operator.keys == 5);
-        assert(operator.limit == 5);
-
-        operatorsRegistry.addValidators(index, 10, tenKeys);
-        operator = operatorsRegistry.getOperator(index);
-        assert(operator.keys == 15);
-    }
```

Comment:
@iamsahu why did we delete these tests ? I would have imagined that business logic shouldn't have changed in this PR ?

Reply:
The test has been left out incorrectly. I have fixed it in the next PR. There was also a bug introduced due to incorrect suggestion from report being implemented.